### PR TITLE
feat(add a template for issues): provide new option for metrics

### PR DESCRIPTION
**Pull request by smerle33**

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/5-metrics.yml [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  - type: input
+    attributes:
+      label: Jenkins Agent Name
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        you can provide information about your built too, that will help providing accurate informations
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/5-metrics.yml:19]:
```suggestion
        you can provide information about your build too, that will help providing accurate informations
```

File: .github/ISSUE_TEMPLATE/5-metrics.yml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  - type: textarea
+    attributes:
+      label: build link or informations
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/5-metrics.yml:22]:
```suggestion
      label: Build link or information
```

File: .github/ISSUE_TEMPLATE/5-metrics.yml [hunk 3]
```diff
@@ -0,0 +1,24 @@
+name: '📊 Infra Specific agent metrics'
+labels: ['metrics', 'triage']
+description: Request the metrics concerning a specific jenkins agent from ci.jenkins.io.
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        the jenkins agent name from the built logs.
```
Comment (timja) [inline .github/ISSUE_TEMPLATE/5-metrics.yml:9]:
```suggestion
        the Jenkins agent name from the build logs.
```

File: .github/ISSUE_TEMPLATE/5-metrics.yml [hunk 4]
```diff
@@ -0,0 +1,24 @@
+name: '📊 Infra Specific agent metrics'
+labels: ['metrics', 'triage']
+description: Request the metrics concerning a specific jenkins agent from ci.jenkins.io.
```
Comment (lemeurherve) [inline .github/ISSUE_TEMPLATE/5-metrics.yml:3]:
```suggestion
description: Request metrics concerning a specific jenkins agent from ci.jenkins.io.
```

File: .github/ISSUE_TEMPLATE/5-metrics.yml [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+
+body:
+  - type: markdown
+    attributes:
+      value: |
+        the Jenkins agent name from the build logs.
```
Comment (lemeurherve) [inline .github/ISSUE_TEMPLATE/5-metrics.yml:9]:
```suggestion
        The Jenkins agent name from the build logs.
```

File: .github/ISSUE_TEMPLATE/5-metrics.yml [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+  - type: input
+    attributes:
+      label: Jenkins Agent Name
+    validations:
+      required: true
+
+  - type: markdown
+    attributes:
+      value: |
+        you can provide information about your build too, that will help providing accurate informations
```
Comment (lemeurherve) [inline .github/ISSUE_TEMPLATE/5-metrics.yml:19]:
```suggestion
        You can provide a link and/or information about your build too, that will help providing accurate informations.
```

File: README.md [hunk 7]
```diff
@@ -2,13 +2,14 @@
...
 - [📝 Documentation](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=documentation&template=3-documentation.yml): Let us know if any documentation is missing or could be improved
-- [🌐 Crowdin localization projects](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=crowdin&template=4-crowdin.yml): Request the creation of new projects on crowdin.jenkins.io (to localize a plugin) or update its members  
+- [🌐 Crowdin localization projects](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=crowdin&template=4-crowdin.yml): Request the creation of new projects on crowdin.jenkins.io (to localize a plugin) or update its members
 - [💬 Community forum](https://community.jenkins.io/): Please ask and answer questions here
 - [✉️ Mailing lists](https://www.jenkins.io/mailing-lists/): You can also raise a question on the infra mailing list
+- [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml) : if you want to get information about the metrics on a specific agent after a built
```
Comment (lemeurherve) [inline README.md:12]:
```suggestion
- [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml): Request infrastructure specific metrics on a Jenkins agent after a build
```
What "after a build" is for there? Is it necessary?

File: README.md [hunk 8]
```diff
@@ -2,13 +2,14 @@
...
 - [📝 Documentation](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=documentation&template=3-documentation.yml): Let us know if any documentation is missing or could be improved
-- [🌐 Crowdin localization projects](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=crowdin&template=4-crowdin.yml): Request the creation of new projects on crowdin.jenkins.io (to localize a plugin) or update its members  
+- [🌐 Crowdin localization projects](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=crowdin&template=4-crowdin.yml): Request the creation of new projects on crowdin.jenkins.io (to localize a plugin) or update its members
 - [💬 Community forum](https://community.jenkins.io/): Please ask and answer questions here
 - [✉️ Mailing lists](https://www.jenkins.io/mailing-lists/): You can also raise a question on the infra mailing list
+- [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml): Request infrastructure specific metrics on a Jenkins agent after a build
```
Comment (lemeurherve) [inline README.md:12]:
```suggestion
- [📊 Infra Specific agent metrics](https://github.com/jenkins-infra/helpdesk/issues/new?assignees=&labels=inframetrics&template=5-metrics.yml): Request infrastructure specific metrics on a Jenkins agent to monitor a build by yourself
```
(nit)

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**